components/instructions/programs/dual.tsx

```diff
@@ -8,6 +8,7 @@ import { AccountMetaData } from '@solana/spl-governance'
 import { tryGetMint } from '@utils/tokens'
 import { getMintDecimalAmountFromNatural } from '@tools/sdk/units'
 
+// TODO: Update with configV3 and initStrikeReversible
```

Comment:
What are the differences here?

Reply:
Different prefix for the instr.

Will have same arguments except an additional mint/token account for the reverse side.

Note that this is only for the helper text, doesnt impact the actual tx